### ISSUE TITLE
A11y_VisualStudioCodeClient_Welcome Screen_ScreenReader: Control type is not announced as button for the action items under walkthrough.

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#141700]
### GitHubTags:

#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Visual Studio Code Client; #BM-VisualStudioCode-Win32-Nov2021; #Win32; #Narrator;#NVDA; #Element:Button; #ScreenReader; #NameRoleValue #WCAG4.1.2; #AINotInScope; #FTP; #STP;

### Environment Details:
Application Name: Visual Studio Code Client
Windows Version: Windows 11

### Repro Steps:

1)Turn on Narrtor/NVDA
2) Open Visual studio code editor.
3) Using "Alt+H" key navigate to the "Help" menu bar option .
4) Using "Down" arrow key navigate to the "Welcome" item.
5) Make sure you have the following  setting in settings.json: "workbench.startupEditor": "gettingStarted"
6) Using "Alt+H" navigate to "Help" menu bar option and use "down" arrow key to navigate to "Getting started". 
7) Verify that whether screen reader is announcing the actionable items under walkthrough as button or not.

### Actual Result:
Control type is not announced as button for the action items under walkthrough.

### Expected Result:
Control type should be announced as button for the action items under walkthrough.

### User Impact:
Screen reader user will face difficulty in finding the functionality.
![#141700_A11y_VisualStudioCodeClient_Welcome Screen_ScreenReader](https://user-images.githubusercontent.com/98530069/151556816-e1174043-e44f-41e1-922a-4857dd0ebac8.gif)

**Comment by JacksonKearl** [comment]
@isidorn I'm using proper HTML `button`s, but I've set the `role` to `listitem` as part of a different accessibility ticket. Do you know what the right approach is here?

**Comment by isidorn** [comment]
Role should be button. That other ticket was probably invalid.
So let's go with button :)

**Comment by JacksonKearl** [comment]
\closedWith dc83d7ebff13ef16fb3508d55e96294e359e6542
